ci-operator/config/amq-streams-qe/strimzi-kafka-operator/amq-streams-qe-strimzi-kafka-operator-main__amq-streams-ocp4.13-lp-interop.yaml

```diff
@@ -0,0 +1,55 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- dockerfile_literal: |
+    FROM quay.io/strimzi-qe/strimzi-tests:release-0.32.x
+    WORKDIR /home/jenkins/interop
+    CMD ["/bin/bash"]
+  to: test-image
+releases:
+  initial:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: amq-streams-2.3
+  cron: 0 1 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: aws.interop.ccitredhat.com
+      GROUPS: sanity
+      OLM_OPERATOR_CHANNEL: stable
```

Comment:
How are you installing the operator? Is it from a script in the test repository? We can use the [`install-operators` ref](https://github.com/openshift/release/tree/master/ci-operator/step-registry/install-operators). It installs an operator using the Operator Hub and allows you to specify as many operators as you'd like.

Reply:
We install operator in our test suite via OperatorHub as well. However, our test suite doesn't support the option to run against an already existing operator and we don't have the capacity to implement it right now. We agreed that we will use our current way.

Reply:
Hi @calebevans , we all agreed with the solution that we will follow the PQE's test suite for the operator installation.

Reply:
@Sau1506mya @Frawless No problem! I think I remember that conversation now, was just curious.